lib/vm.py

```diff
@@ -12,8 +12,11 @@
 from lib.snapshot import Snapshot
 from lib.vif import VIF
 
+if TYPE_CHECKING:
+    from lib.host import Host
```

Comment:
```suggestion
    import lib.host
```

Reply:
Why?  I noticed you did this in the previous branch, but it deviates from the use we use the classes outside of typing context, and if makes the hints themselves IMO-unnecessarily longer.